scripts/pylib/pytest-twister-harness/src/twister_harness/device/qemu_adapter.py

```diff
@@ -20,7 +18,8 @@
 class QemuAdapter(BinaryAdapterBase):
     def __init__(self, device_config: DeviceConfig, **kwargs) -> None:
         super().__init__(device_config, **kwargs)
-        fifo_path = Path(self.device_config.build_dir) / QEMU_FIFO_FILE_NAME
+        qemu_fifo_file_name = 'qemu-fifo'
+        fifo_path = self.device_config.build_dir / qemu_fifo_file_name
         self._fifo_connection: FifoHandler = FifoHandler(fifo_path, self.base_timeout)
 
```

Comment:
```suggestion
self._fifo_connection: FifoHandler = FifoHandler(
            self.device_config.build_dir / 'qemu-fifo',
            self.base_timeout )
```
po co dodawać extra linie ze zmiennymi, jeśli nie są one nigdzie dalej używane?